src/core/textures/Texture.js

```diff
@@ -143,6 +144,13 @@ export default class Texture extends EventEmitter
             baseTexture.once('loaded', this.onBaseTextureLoaded, this);
         }
 
+        /**
+         * Anchor point that is used as default if sprite is created with this texture
```

Comment:
Since this is not an observable point, it should be noted that changing values to this point do not affect Sprites that reference this texture. That could confuse people.